src/main/java/com/spotify/reaper/resources/RepairRunResource.java

```diff
@@ -161,12 +161,10 @@ public Response addRepairRun(
         parallelism = RepairParallelism.valueOf(repairParallelism.get().toUpperCase());
       }
       
-      if(!parallelism.equals(RepairParallelism.PARALLEL) && incrementalRepair) {
-          return Response.status(Response.Status.BAD_REQUEST).entity(
-                  "It is not possible to mix sequential repair and incremental repairs. parallelism " 
-                  + parallelism + " : incrementalRepair " + incrementalRepair).build();
+      if (incrementalRepair) {
+        parallelism = RepairParallelism.PARALLEL;
```

Comment:
I feel like if PARALLEL is the only option for incremental repair, we should just force it instead of asking for the user to pick the right one.
Furthermore, the UI doesn't require this field to be set and then SEQUENTIAL is selected as the default, which is not compatible with inc repair.